README.md

```diff
@@ -236,24 +265,77 @@ Type: `number`
 
 **Required: ✓**
 
-The width of collection
+The width of collection viewport
 
 ### height
 
 Type: `number`
 
 **Required: ✓**
 
-The height of collection
+The height of collection viewport
+
+### scrollToBottomRange
+
+Type: `number`
+
+When present the component will emit `scrolled-to-bottom-range` when the bottom is >= 1 and the number provided.
+The `scrolled-to-bottom` will still be fired and the 2 events will not be emitted at the same time.
+
+### containerPaddingBottom
+
+Type: `number`
+
+Optionally extend the calculated height of the collection container, the affect is apparent padding-bottom
+
+### headerSlotHeight
+
+Type: `number`
+
+When injecting content into the header slot, you should also add the px height of the slot. This ensure items are removed from view at the right time.
```

Comment:
Could you note default value of these props?

Reply:
updated